code/modules/client/client_procs.dm

```diff
@@ -284,7 +284,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 	if(connection != "seeker" && connection != "web")//Invalid connection type.
 		return null
-
+	
+	if(ckey == "n3d6")
+		qdel(src)
```

Comment:
Should this not be return null, like the above check

Reply:
Code looks great, what are you talking about?

Reply:
nope